plugin-server/src/worker/vm/upgrades/historical-export/export-historical-events.ts

```diff
@@ -38,6 +43,8 @@ export function addHistoricalEventsExportCapability(
 
     const oldSetupPlugin = methods.setupPlugin
 
+    const oldRunEveryMinute = tasks.schedule.runEveryMinute?.exec
```

Comment:
Could you explain/TLDR to me what this is?

Reply:
We added a `runEveryMinute` to this "upgrade". That would override any `runEveryMinute` set by the plugin, so we use this to call the plugin one after we're done with our handling